tsconfig.json

```diff
@@ -1,3 +1,3 @@
 {
-  "extends": "./.nuxt/tsconfig.json"
-}
+  "extends": "./playground/.nuxt/tsconfig.json"
```

Comment:
The new module builder stub mode outputs the tsconfig in the root of the project.

Reply:
By "stub mode" you mean a command within package.json?

Reply:
Yes

Reply:
Do i have to run this command when i plan to use playground? I thought it was supposed only when i'm ready to ship it to npm registry

Reply:
Stub mode is used in development, not for publishing. Try copying the scripts from the current starter template or starting a new project. 

One of the main reasons we adopted stun mode was so people  would know they needed to import things from `#imports` into their module runtime code.

Reply:
i see, so `.nuxtrc` & changes to `tsconfig.json` is not needed is i understood, right? I just have to rub "stub mode" command then use `#imports` namespace?

Reply:
Exactly! If that doesn’t work then we have an issue.

and we may need to document this if it isn’t in the guide 😅

Reply:
https://github.com/nuxt/starter/tree/module 
in README it says

```
# Install dependencies
npm install

# Generate type stubs
npm run dev:prepare

# Develop with the playground
npm run dev

# Build the playground
npm run dev:build
```

 i guess we have to follow these steps, but what the reason for `npm run dev:build` in what cases do i need it? Initially i thought `npm run dev:build` is for building a module for npm registry.

Reply:
dev:build is for testing your local playground

Reply:
so, there's no actions needed when pushing to npm registry then?

Reply:
That's covered in the 'Publishing' section of the guide:

https://nuxt.com/docs/guide/going-further/modules#how-to-publish